<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SyntacticAnalyzerClass</class>
 <widget class="QMainWindow" name="SyntacticAnalyzerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1017</width>
    <height>490</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>算符优先语法分析器</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_6">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="5,0">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
        <property name="sizeConstraint">
         <enum>QLayout::SetFixedSize</enum>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
          <item>
           <widget class="QGroupBox" name="groupBox">
            <property name="title">
             <string>FIRST和LAST集</string>
            </property>
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QTableView" name="lastView">
               <property name="styleSheet">
                <string notr="true">QHeaderView::section {
    color: white;
    background-color: rgb(0, 170, 255);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(0, 170, 255);
} </string>
               </property>
               <attribute name="horizontalHeaderVisible">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderDefaultSectionSize">
                <number>100</number>
               </attribute>
               <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
               <attribute name="horizontalHeaderStretchLastSection">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderCascadingSectionResizes">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderShowSortIndicator" stdset="0">
                <bool>true</bool>
               </attribute>
               <attribute name="verticalHeaderStretchLastSection">
                <bool>false</bool>
               </attribute>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>优先关系表</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QTableView" name="tableView">
             <property name="styleSheet">
              <string notr="true">QHeaderView::section {
    color: white;
    background-color: rgb(0, 170, 255);
    border: 5px solid #f6f7fa;
    border-radius:0px;
    border-color: rgb(0, 170, 255);
} </string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>输入项</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <widget class="QTextEdit" name="input_text">
             <property name="textInteractionFlags">
              <set>Qt::TextSelectableByKeyboard|Qt::TextSelectableByMouse</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>检测结果</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <layout class="QVBoxLayout" name="output_verticalLayout">
             <item>
              <widget class="QTextBrowser" name="output_text"/>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1017</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="open_exp"/>
   <addaction name="check"/>
   <addaction name="input_grammars"/>
   <addaction name="input_treminal"/>
   <addaction name="input_exp"/>
   <addaction name="read_settings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="open_exp">
   <property name="text">
    <string>导入表达式</string>
   </property>
   <property name="toolTip">
    <string>导入表达式</string>
   </property>
  </action>
  <action name="check">
   <property name="text">
    <string>检查</string>
   </property>
   <property name="toolTip">
    <string>检查</string>
   </property>
  </action>
  <action name="input_grammars">
   <property name="text">
    <string>输入文法</string>
   </property>
   <property name="toolTip">
    <string>输入文法</string>
   </property>
  </action>
  <action name="input_treminal">
   <property name="text">
    <string>输入终结符</string>
   </property>
  </action>
  <action name="input_exp">
   <property name="text">
    <string>输入表达式</string>
   </property>
  </action>
  <action name="read_settings">
   <property name="text">
    <string>配置</string>
   </property>
   <property name="toolTip">
    <string>配置</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SyntacticAnalyzer.qrc"/>
 </resources>
 <connections/>
</ui>
